content/en/continuous_integration/tests/setup/ruby.md

```diff
@@ -260,6 +264,36 @@ The following environment variable can be used to configure the location of the
 
 All other [Datadog Tracer configuration][6] options can also be used.
 
+## Using additional instrumentations
+
+It can be useful to have rich tracing information about your tests that includes time spent performing database operations
+or other external calls like here:
```

Comment:
```suggestion
It can be useful to have rich tracing information about your tests that includes time spent performing database operations or other external calls, as seen in the following flame graph:
```